boxsdk/util/url.py

```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+
+from __future__ import unicode_literals, absolute_import
+
+from boxsdk.config import API
+
+
+def get_url(endpoint, *args):
```

Comment:
I have an in-progress change that changes how we use the ``API`` configs. Having this global function would not mesh well with what I had in mind.

Can we either make this private (so that we don't have to worry about future backwards-compatibility issues), or move it to be an instance method on ``BoxSession`` (where it can be used by ``BaseEndpoint`` and ``Client``)?